apps/docs/stories/components/CardForm/index.stories.tsx

```diff
@@ -0,0 +1,162 @@
+import type { Meta } from "@storybook/web-components";
+import { withActions } from "@storybook/addon-actions/decorator";
+import {
+  CSSVarsExample,
+  StoryBaseArgs,
+  paymentMethodFormComponentMethods,
+} from "../../utils";
+
+import "@justifi/webcomponents/dist/module/justifi-card-form";
+
+const storyBaseArgs = new StoryBaseArgs([
+  "account-id",
+  "client-id",
+  "iframe-origin",
+  "theme",
+]);
+storyBaseArgs.argTypes["client-id"].table.disable = true;
```

Comment:
Is this being hidden in favor of web component tokens?

Reply:
Actually, this code was already there before in main. Not sure why either.